var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -75,7 +75,7 @@ class Exago(CMakePackage, CudaPackage, ROCmPackage):
     depends_on('hiop+mpi', when='+hiop+mpi')
 
     depends_on('petsc@3.13:3.14', when='@:1.2.99')
-    depends_on('petsc@3.16.0:', when='@1.3.0:')
+    depends_on('petsc@3.16:3.16', when='@1.3.0:')
```

Comment:
Assuming this works (as opposed to 3.16.0:3.16), LGTM

Reply:
`3.16:3.16` does work, I tested it :)

Reply:
Fantastic, thanks

Reply:
the safer option is `3.16.0:3.16` since `3.16:3.16` "simplifies" due to a bug in Spack to `3.16`.